.github/workflows/main.yml

```diff
@@ -20,7 +20,7 @@ jobs:
           node-version: ${{ matrix.node }}
 
       - name: Install deps and build (with cache)
-        uses: bahmutov/npm-install@v1
+        uses: bahmutov/npm-install@c67aaab58a864ea2873950cde9c1c9379f9f711a
```

Comment:
What's this change?

Reply:
Since this is a third-party action, it's safer to use the commit hash instead of a tag.

https://docs.github.com/en/actions/learn-github-actions/security-hardening-for-github-actions#using-third-party-actions